packages/cfa-template/template/frontend/.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-    "extends": "@friggframework/eslint-config"
+//    "extends": "@friggframework/eslint-config"
```

Comment:
Reason for this?

Reply:
Still did not check why but it's throwing an error during build